airbyte-integrations/connectors/destination-s3-v2/src/main/kotlin/S3V2Configuration.kt

```diff
@@ -52,7 +51,12 @@ class S3V2ConfigurationFactory :
             s3BucketConfiguration = pojo.toS3BucketConfiguration(),
             objectStoragePathConfiguration = pojo.toObjectStoragePathConfiguration(),
             objectStorageFormatConfiguration = pojo.toObjectStorageFormatConfiguration(),
-            objectStorageCompressionConfiguration = pojo.toCompressionConfiguration()
+            objectStorageCompressionConfiguration = pojo.toCompressionConfiguration(),
+            objectStorageUploadConfiguration =
+                ObjectStorageUploadConfiguration(
+                    pojo.uploadPartSize ?: (5L * 1024 * 1024),
+                    pojo.maxConcurrentUploads ?: 1
```

Comment:
Nit: should defaults be consts somewhere?

Reply:
Done. Added to UploadConfiguration